src/components/SwitchToggle.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import PropTypes from 'prop-types';
+
+const Checkbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+`;
+
+const Label = styled.label`
+  align-items: center;
+  display: flex;
+  min-height: 2rem;
+  padding-left: 3.5rem;
+  position: relative;
+  user-select: none;
+  &::before {
+    background-color: #fff;
+    border: 1px solid
+      ${({ checked, theme }) => (checked ? theme.button : '#ece9f1')};
+    border-radius: 100rem;
+    box-shadow: ${({ theme }) => theme.boxShadowSmall};
+    content: '';
+    display: block;
+    height: 1.75rem;
+    left: 0;
+    position: absolute;
+    top: 0.125rem;
+    transition: border-color ${({ theme }) => theme.transitionEase};
+    width: 3rem;
+  }
+  &::after {
+    background-color: ${({ checked, theme }) =>
+      checked ? theme.button : '#ece9f1'};
```

Comment:
Same as above with the border. Instead of `theme.button` you can use `theme.border` and instead of the hardcoded hex code, you can use `theme.borderActive`.

Reply:
done )